.github/workflows/codeql.yml

```diff
@@ -0,0 +1,37 @@
+name: CodeQL - Code scanning
+
+on:
+  schedule:
+    - cron: '0 10 * * 1'
+
+jobs:
+  codeql-scan:
+
+    strategy:
+      fail-fast: false
+
+    runs-on: macos-latest
+
+    permissions:
+      security-events: write
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
```

Comment:
The "Android CI" workflow uses `zulu`. Is there an advantage to using different ones or should we stick to one?

Reply:
I guess we should stick to one, `temurin` is the default suggested version from GitHub and is provided by [Adoptium](https://adoptium.net/de/). `zulu` is new to me and according to "[Which Version of JDK Should I Use?](https://whichjdk.com/#azul-zulu)" also recommended. Both are fully compatible and provide also x86 and arm versions. So the only difference it who's providing it.

Reply:
Can you then please create another PR changing the other workflow to use `temurin`?

Reply:
Sure